ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,82 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def check_mirroring_status(replaying_num=None):
+    """
+    Check mirroring status have expected values
+
+    Args:
+        replaying_num (int): Expected number of images in replaying state
+
+    Returns:
+        bool: True if status contains expected values, False otherwise
```

Comment:
Same as above, please specify the expected values

Reply:
updated